hbase-thrift/pom.xml

```diff
@@ -444,6 +444,7 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-minicluster</artifactId>
+          <scope>test</scope>
```

Comment:
During work for this PR, found that a lot of test jars were coming to our assembly as hadoop-minicluster was marked as compile scope. Thinking of doing this as another JIRA to keep PR simpler.

Reply:
Any thoughts? @stoty

Reply:
Now when I think again, seems it is better this way.

Reply:
Marking as resolved!